evalml/tests/automl_tests/test_automl.py

```diff
@@ -702,8 +709,10 @@ def test_invalid_data_splitter(X_y_binary):
 
 
 def test_large_dataset_binary(AutoMLTestEnv):
-    X = pd.DataFrame({"col_0": [i for i in range(101000)]})
-    y = pd.Series([i % 2 for i in range(101000)])
+    X = pd.DataFrame(
+        {"col_0": [i for i in range(111113)]}
```

Comment:
Nitpick: 100% not a you thing (pretty sure I do the same thing), but we can simplify `[i for i in range(x)]` to just be `range(x)`

Reply:
Moved check to `use_holdout`